packages/esm-patient-medications-app/src/order-basket/order-basket-search/order-basket-search-results.component.tsx

```diff
@@ -13,16 +13,32 @@ export interface OrderBasketSearchResultsProps {
   searchTerm: string;
   setSearchTerm: (value: string) => void;
   onSearchResultClicked: (searchResult: OrderBasketItem, directlyAddToBasket: boolean) => void;
+  searchInputRef: RefObject<HTMLInputElement>;
 }
 
 export default function OrderBasketSearchResults({
+  searchInputRef,
   searchTerm,
   setSearchTerm,
   onSearchResultClicked,
 }: OrderBasketSearchResultsProps) {
   const { t } = useTranslation();
   const isTablet = useLayoutType() === 'tablet';
   const { drugs, isLoading, error } = useDrugSearch(searchTerm);
+  const [isSearchIputCleared, setIsSearchInputCleared] = useState(false);
+  function focusOnSearchInput() {
+    searchInputRef.current.focus();
+  }
+  function handleClick() {
+    setIsSearchInputCleared(true);
+    setSearchTerm('');
+    focusOnSearchInput();
+    handleClearSearchInput();
```

Comment:
This function is doing what the above 2 lines are doing
Duplication of code
You can delete this function altogether